src/utils/tokenMetadata.ts

```diff
@@ -28,13 +28,18 @@ function hex_to_ascii(str1: string) {
 }
 
 function createGetMultiAssetTxMintMetadataQuery(assets: PolicyIdAssetMapType) {
+  // each asset (policy + assetName pair) gets passed as a SQL argument
+  // so we need to give each one of these an argument number in postgres
+  // note: postgres has a max of 65535 (so 32K pairs)
+  // but it's unlikely somebody passes 32K pairs in a single request to the backend
+  let index = 1;
```

Comment:
isnt this suppose to start from 0?

Reply:
No, SQL arguments start at index 1. See the "before" as well

Reply:
I can add a comment to remind people of this since it isn't intuitive